scripts/lib/CIME/SystemTests/system_tests_compare_two.py

```diff
@@ -184,8 +185,9 @@ def build_phase(self, sharedlib_only=False, model_only=False):
             # Although we're doing separate builds, it still makes sense
             # to share the sharedlibroot area with case1 so we can reuse
             # pieces of the build from there.
-            self._case2.set_value("SHAREDLIBROOT",
-                                  self._case1.get_value("SHAREDLIBROOT"))
+            if get_model() != "acme":
```

Comment:
Can you please add a comment describing why it doesn't work to do this for acme?

Reply:
Is this a deficiency in the mpas build that should be corrected?

Reply:
Yes, at some point it will need to be fixed. I've noticed that ACME doesn't do well with sharedlibs in general, which is why we still force every case to build its own sharedlibs.